dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile

```diff
@@ -8,5 +8,7 @@ USER root
 # Install curl and bash
 # Install ssh for cloning ssh-repositories
 # Install less for handling git diff properly
-RUN yum install -y sudo git bzip2 which bash curl openssh less && \
-    yum -y clean all && rm -rf /var/cache/yum
+# Install sshpass for handling passwordds for SSH keys
+RUN yum install -y sudo git bzip2 which bash curl openssh less \
+    https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm && \
+    yum --enablerepo="epel" install -y sshpass && yum -y clean all && rm -rf /var/cache/yum
```

Comment:
this definitely won't work in Brew but I can probably (?) find a workaround. If you are able to contribute this same fix in downstream that would be super helpful. https://github.com/redhat-developer/codeready-workspaces-theia/blob/master/conf/theia/ubi8-brew/runtime-install-dependencies.dockerfile

Reply:
https://github.com/redhat-developer/codeready-workspaces-theia/pull/14

Reply:
As I said we can't depend on EPEL in downstream. EPEL is not available in Brew.

Reply:
See comments on redhat-developer/codeready-workspaces-theia#14 -- best option here would be to pull the sources and store them so that the `ubi8-brew/runtime-install-dependencies.dockerfile` script can copy them into the docker context and unpack & use them